DEVELOPMENT.md

```diff
@@ -0,0 +1,18 @@
+# Development
+
+This repository contains the scripts and patches used to build and download the Node.js binaries for pkg.
+
+## New releases
+
+Pkg will check for releases of pkg-fetch `<major>.<minor>` version. So each minor/major release of pkg-fetch requires a new GH release with all attached binaries. Patch releases don't require to recreate all new binaries, but only the ones that have changed.
+
+Important things:
+
+- In order to support a new nodejs version we need to create/update the patch in `patches` folder and also `patches/patches.json` file.
+- Once the new patch is merged we can trigger the `.github/workflows/build-all.yml` workflow to build all the binaries for all the platforms.
+- If this is a minor/major release we need to check the checkbox of the workflow to create a draft release.
+- Once the actions will end, copy the sha256 checksums printed at the end of the release body and update`lib/expected-shas.json` file with the correct values.
```

Comment:
I thought that the workflow itself posts a summary with the SHAs so they are no longer in the (mutable) release, but rather attached to the workflow that is seemingly immutable. I don't know.

Reply:
So expected sha are no longer needed? I need to check the code so... I forgot about this

Reply:
I confirm I don't see any sha chech in the code except for the nodejs tar, btw we are still producing them, what am I missing here? 🤔

Reply:
I see there is a script in package.json named `updateExpected` but that is never called in the buildall process. I think that should be manually run and by passing the generated sha in the workflow release: 

https://github.com/yao-pkg/pkg-fetch/blob/8015eb2f02b59b911a22b7cff78aceb8cad46148/lib/generate-expected-shas.ts#L10